ansible/site.yml

```diff
@@ -10,3 +10,8 @@
 - import_playbook: nessus.yml
 - import_playbook: newrelic-infrastructure.yml
 - import_playbook: unattended-upgrades.yml
+
+
+# App plays
+############
+- import_playbook: catalog.yml
```

Comment:
Was this meant to be included? I'm deleting the site.yml playbook altogether with the refactoring work